Fix Issue#36

ChristoferVikstroem opened the pull request:
Fixed the division by zero bug, added documentation and relevant test cases.

ChristoferVikstroem committed:
Fix Issue#36

Fixed the division by zero bug, added documentation and relevant test cases.

OudayAhmed commented on src/cmv.py:
```diff
@@ -186,28 +186,45 @@ def cmv5(NUMPOINTS, POINTS):
+                    side_2 = math.sqrt((POINTS[i + j][0] - POINTS[i][0]) ** 2 + (POINTS[i + j][1] - POINTS[i][1]) ** 2)
+                    side_3 = math.sqrt((POINTS[i + N_PTS - 1][0] - POINTS[i + j][0]) ** 2 + (
+                            POINTS[i + N_PTS - 1][1] - POINTS[i + j][1]) ** 2)
+                    if side_1 == 0.0 or side_2 == 0.0 or side_3 == 0.0:
```
I think you should not return false when you are checking if side_1 == 0.0 or side_2 == 0.0 or side_3 == 0.0. Instead of returning false, you can maybe use continue.

ChristoferVikstroem replied:
Good catch! 
I changed it to continue instead of returning False.

ChristoferVikstroem committed:
Fix Issue/36

Changed "return false" to "continue" in method cmv6 as suggested by the reviewer!

OudayAhmed commented:
LGTM!
